src/main/webapp/script.js

```diff
@@ -184,9 +184,17 @@ function createListElement(text) {
   return liElement;
 }
 
+var shareShowing = Boolean(false)
 function showShare() {
-    console.log("Show");
-    document.getElementById("share-form").classList.remove("invisible");
+    if (!shareShowing){
+        shareShowing = true;
```

Comment:
One small clean-up would be to do shareShowing = !shareShowing after the if-else statement instead of shareShowing = true and shareShowing = false.

Reply:
Done.